types/prosemirror-view/index.d.ts

```diff
@@ -180,6 +180,8 @@ export class EditorView {
    * Focus the editor.
    */
   focus(): void;
+
+  focused: boolean;
```

Comment:
> It is not mentioned in docs, but this prop is actually the only

This means that isn’t a private API and should not be relied on. As such it’s not appropriate to add to the types. 

If you’d like this API made public please raise an issue on prosemirror/prosemirror.

Reply:
Isn’t a public API**

Reply:
Makes sense, thanks!